Python/libraries/recognizers-date-time/recognizers_date_time/date_time/base_date.py

```diff
@@ -302,6 +302,20 @@ def validate_match(self, match: Match, text: str):
                     sub_text = text[text.index(year_group) + len(year_group):].strip()
                     sub_text = self.trim_start_range_connector_symbols(sub_text)
                     is_valid_match = self.starts_with_basic_date(sub_text)
+                    
+            # Expressions with mixed separators are not considered valid dates e.g. "30/4.85" (unless one is a comma "30/4, 2016")
```

Comment:
This should be a method, like in .NET and Java, right?

Reply:
also in Java and .NET, I added this check as a further step in the already existing method ValidateMatch